examples/graphs/node2vec_movielens.py

```diff
@@ -547,7 +547,7 @@ def create_model(vocabulary_size, embedding_dim):
 indices = indices.numpy().tolist()
 
 """
-Display the top related movies the query movies
+Display the top related movies the query movies.
```

Comment:
I think this should be rephrased. Currently, it is challenging to understand what "Display the ... movies the ... movies" means.

Suggestion:

- Define in non-code language what `query_movies` is about (the <thing>).
- Rephrase the sentence to something like "Display the top related movies from the <thing>.